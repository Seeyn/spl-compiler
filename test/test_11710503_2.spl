struct foo{
    int a;
    float b,c,d;
};

int main(){
    int e[10];
    float f = 1.0;
    struct foo g;
    g.b = f;
    e[0] = f;
    g.b[1] = f; 
}