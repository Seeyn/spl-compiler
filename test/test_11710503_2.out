Error type 5 at Line 11: unmatching type on both sides of assignment
Error type 10 at Line 12: indexing on non-array variable
